### PR TITLE
[a11y] com_contenthistory on mobile devices

### PR DESCRIPTION
**Pull request by brianteeman**
Recent changes have aimed to improve the accessibility where we use font icons to ensure that their value is not "read" by assistive technology and only the associated text is read.

The buttons in the version history modal are not constructed the same as other buttons in Joomla and when we switch to a mobile device the displayed text is hidden.

This means that with the current markup it is all ok for accessibility on a desktop because we have the associated text but as this text is hidden on a phone the assistive technology has no way of identifying the buttons.

This PR adds the aria-label attribute to the buttons to ensure that assistive technology can always "read" the button at all screensizes.

Thanks to @fuzzbomb for spotting this and his guidance on resolving this

### CODE REVIEW

File: administrator/components/com_contenthistory/views/history/tmpl/modal.php [hunk 1]
```diff
@@ -91,15 +91,15 @@
...
+		<button onclick="if (document.adminForm.boxchecked.value==0){<?php echo $deleteMessage; ?>}else{ Joomla.submitbutton('history.keep')}" class="btn pointer hasTooltip" aria-label="<?php echo JText::_('COM_CONTENTHISTORY_BUTTON_KEEP_DESC'); ?>" title="<?php echo JText::_('COM_CONTENTHISTORY_BUTTON_KEEP_DESC'); ?>">
 			<span class="icon-lock" aria-hidden="true"></span><span class="hidden-phone"><?php echo JText::_('COM_CONTENTHISTORY_BUTTON_KEEP'); ?></span></button>
-		<button onclick="if (document.adminForm.boxchecked.value==0){<?php echo $deleteMessage; ?>}else{ Joomla.submitbutton('history.delete')}" class="btn pointer hasTooltip" title="<?php echo JText::_('COM_CONTENTHISTORY_BUTTON_DELETE_DESC'); ?>">
+		<button onclick="if (document.adminForm.boxchecked.value==0){<?php echo $deleteMessage; ?>}else{ Joomla.submitbutton('history.delete')}" class="btn pointer hasTooltip" taria-label="<?php echo JText::_('COM_CONTENTHISTORY_BUTTON_DELETE_DESC'); ?>" title="<?php echo JText::_('COM_CONTENTHISTORY_BUTTON_DELETE_DESC'); ?>">
```
Comment (Quy) [inline administrator/components/com_contenthistory/views/history/tmpl/modal.php]: Remove the `t` before `aria`.

Comment (brianteeman) [inline administrator/components/com_contenthistory/views/history/tmpl/modal.php]: i need new glasses - thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3a3f51b6f30f7b311293d222ad9aae7e668d5e72<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15828">issues.joomla.org/tracker/joomla-cms/15828</a>.</sub>

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 3a3f51b6f30f7b311293d222ad9aae7e668d5e72<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15828">issues.joomla.org/tracker/joomla-cms/15828</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**